### DISCUSSION TITLE
Agregar un mensaje de "se ha enviado el fomrulario o gracias por enviar" cuando se utiliza los fomrularios públicos

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#103] [3. Ideas & Feature Requests] [answered]
Hola

Me percaté que no arroja un mensaje de salida cuando se envía un formulario público, creo que sería bueno agregar un mensaje. Quizás está pero no se ve.

Adjunto imagen de referencia
![image](https://user-images.githubusercontent.com/61419471/115736057-fbb00b00-a358-11eb-8947-bf876123488a.png)

**Comment by acedward** [comment]
Es configurable, en el próximo release.

**Comment by acedward** [comment, marked as answer]
@NicolasPLG plz espera el anuncio en Whats new! 
Aún es una funcionalidad en desarrollo!
 
https://doc.cotalker.com/docs/updates/new_features

**Reply by NicolasPLG** [reply]
gracias @acedward por la respuesta
